### PR TITLE
Condense settings and properties tabs into one

### LINKED ISSUE #1880: Condense "Settings" and "Properties" tabs into one
After discussion, we have decided to merge these two tabs. The new tab will be named "Settings".

Screenshot 2023-06-29 at 2 46 04 PM

This will leave space for a 3rd tab for Interactions, that Darin is designing. If we want a 4th tab in the future we will change the labels to icons rather than text - no problem.

- [x] Please move content from the Settings tab to the top of the current Properties tab
- [x] Then rename it to "Settings" and delete the old Settings tab.

I changed the names on the Panel Tabs List component to reflect this update
https://www.figma.com/file/sfCE7iLS0k25qCxiifQNLE/%F0%9F%93%9A-Webstudio-Library?type=design&node-id=2647%3A9488&mode=design&t=SNS5w31RSdoJQ8IB-1

### PR DESCRIPTION
**Pull request by kof**
Closes #1880
Mostly also #1905

## Description

Merging Settings and properties

## Code Review

- [ ] hi @TrySound , I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

**Comment by kof** [comment]
@TrySound  I didn't move State to a separate section like Properties, I think its better if you do this

File: apps/builder/app/builder/features/settings-panel/settings-section/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from "./settings-section";
```
Comment (TrySound) [inline apps/builder/app/builder/features/settings-panel/settings-section/index.ts:1]: too many index.ts

File: apps/builder/app/builder/features/settings-panel/props-section/use-props-logic.ts [hunk 2]
```diff
@@ -118,15 +118,6 @@ const getAndDelete = <Value>(map: Map<string, Value>, key: string) => {
...
 const systemPropsMeta: { name: string; meta: PropMeta }[] = [
-  {
-    name: "id",
-    meta: {
-      label: "ID",
-      required: false,
-      control: "text",
-      type: "string",
-    },
-  },
```
Comment (TrySound) [inline apps/builder/app/builder/features/settings-panel/props-section/use-props-logic.ts:129]: could've done in separate PR but ok

**Comment by TrySound** [comment]
Input got unnecessary padding 
image

**Comment by kof** [comment]
@TrySound  this is what I see on preview and locally

Screenshot 2023-07-09 at 22 21 55

File: apps/builder/app/builder/features/settings-panel/settings-section/index.ts (hunk 1 above)
Comment (kof) [reply apps/builder/app/builder/features/settings-panel/settings-section/index.ts:1]: fixing

**Comment by TrySound** [comment]
alright, broken in safari

**Comment by kof** [comment]
@TrySound patched up safari a bit, layout still has issues three, but I have no desire to invest time into it right now

**Review by TrySound — APPROVED** [review]
